go/teams/seitan_test.go

```diff
@@ -149,9 +149,6 @@ func TestSeitanKnownSamples(t *testing.T) {
 	require.Equal(t, "Edwin Powell Hubble", labelSms.F)
 	require.Equal(t, "+48123ZZ3045", labelSms.N)
 
-	// Packing struct is non-deterministic as far as field ordering is
-	// concerned, so we will not be able to get same ciphertext here.
-
```

Comment:
ciphertext is compared at line 160 `require.Equal(t, peiKey.EncryptedIKeyAndLabel, peiKey2.EncryptedIKeyAndLabel)`